frontend/src/pages/Admin/Components/Faq/Q&A/ManageQ&A/ManageQ&A.jsx

```diff
@@ -0,0 +1,159 @@
+import { useEffect, useState } from "react";
+import { END_POINT } from "../../../../../../config/api";
+import style from "./manage.module.scss";
+import { AiOutlineArrowLeft } from "react-icons/ai";
+
+export function Manageqa({ setTab, qId }) {
+  const [ans, setAns] = useState([]);
+  const [qns, setQns] = useState();
+  const [toogle, setToogle] = useState(false);
+  const getQuestion = async (id) => {
+    try {
+      const qUrl = `${END_POINT}/question/getQuestionById`;
+      const qResponse = await fetch(qUrl, {
+        method: "POST",
```

Comment:
it should be a get call, please revert the backend change and refer - https://github.com/HITK-TECH-Community/Community-Website/blob/main/frontend/src/pages/Q%26A/Q%26A.jsx for the api calls